src/components/ResultsMap.vue

```diff
@@ -2,6 +2,7 @@
 	<section>
 		<p>Zoom out on map to ensure all result markers are visible.</p>
 		<GmapMap
+			class="gmap-map"
 			:center="centerMap"
```

Comment:
I was thinking about classes and ids when I was testing and wondering if it is better to add them as a matter of habit or only add them as needed. Like what is the convention or "professional" approach or expectation around that?

Reply:
I prefer to have a class on all of my html elements because it's easier to target those elements if I end up needing to do so.  I don't believe it's required, but it definitely can't hurt.